compileAndBuildVLCKit.sh

```diff
@@ -755,7 +755,10 @@ build_universal_static_lib() {
     arch="aarch64"
     if [ "$FARCH" != "all" ];then
         arch="$FARCH"
+    elif [ "$BUILD_SIMULATOR" = "yes" ]; then
+        arch="x86_64"
```

Comment:
This is inaccurate. There are i686 builds for iOS Simulator.

Reply:
Isn't it the same for non-simulator build then since there are not only aarch64?

Reply:
@fkuehne this arch variable is used to get the list of modules. So, it will works for i686 builds too (like it works for armv7 build when aarch is aarch64).

So, LGTM.